contracts/modules/token/NXMToken.sol

```diff
@@ -218,7 +218,7 @@ contract NXMToken is OZIERC20 {
   * @param value The amount to be transferred.
   */
   function operatorTransfer(address from, uint256 value) public onlyOperator returns (bool) {
-    require(value <= _balances[from]);
+    require(value <= _balances[from], "Insufficient funds to allow transfer");
```

Comment:
The NXMToken contract is immutable and we're "stuck" with it as it is, hence we don't change the code nor compiler version for it. The deps are also copied/frozen.